readme.md

```diff
@@ -231,5 +271,10 @@ Convert the following ActiveRecord sequence to Mongoose:
 
 ```js
 // Your answer...
+db.instructors.find(name: "Andy")
+db.wishlist_items.insert(
+  {
+    wishlist_item: "Resin Laying Deer Figurine, Gold"
+  }
+)
```

Comment:
Not quite... your code wouldn't associate the wish_list item with Andy, and assumes that wishlist_items is a separate table (rather than a nested document on each instructor).

Here's an approach that should work:

```js
Instructor.findOne({name: "Andy"), function(err, result){
  result.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
  result.save();
});
```